README.md

```diff
@@ -5,6 +5,26 @@ These containers allow us to, e.g., not rebuild Trilinos every time we test Pres
 
 When a Dockerfile is modified or added to this repository, the `ci-docker.yml` GitHub workflow will automatically build and push the image to the [Pressio container registry](https://github.com/orgs/Pressio/packages).
 
+Contents:
+- [List of Containers](#list-of-containers)
+- [Adding a New Container](#adding-a-new-container)
+  - [Adding a New Version of Trilinos](#adding-a-new-version-of-trilinos)
+
+## List of Containers
+
+The following table shows the current container configurations maintained by the `pressio-containers` repository.
+
+| OS       | Compiler  | TPL (Optional)                                           |
+| :------- | :-------- | :-------------------------------------------------       |
+| `ubuntu` | `gnu11`   | `mpi` `trilinos` `kokkos-cpu-openmp` `kokkos-cpu-serial` |
+|          | `clang14` | `mpi`                                                    |
+| `fedora` | `gnu13`   | `mpi`                                                    |
+|          | `clang17` | `mpi`                                                    |
```

Comment:
can you please remake this so that each image has it own row? 
 fro example:
 ```
 | OS       | Compiler  | TPL (Optional)        |
| :------- | :-------- | :--------------------- |
| `ubuntu` | `gnu11`   | `mpi` | 
| `ubuntu` | `gnu11`   | `mpi` and `trilinos`  |
```
 i like this better plese